block/manager.go

```diff
@@ -49,6 +51,9 @@ const defaultBlockTime = 1 * time.Second
 // defaultLazyBlockTime is used only if LazyBlockTime is not configured for manager
 const defaultLazyBlockTime = 60 * time.Second
 
+// defaultDABlockTime is used only if DABlockTime is not configured for manager
```

Comment:
```suggestion
// defaultBatchRetrievalInterval is used only if DABlockTime is not configured for manager
```

How can the DABlockTime not be configured? Doesn't it have a default?